app/models/partnership.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Partnership < ApplicationRecord
+  CHALLENGE_WINDOW = 14.days
```

Comment:
Ultra minor - feels a bit odd not to freeze a constant, but I don't really know any way of modifying Duration object.